src/query/plan/operator.cpp

```diff
@@ -3057,6 +3058,47 @@ void EdgeUniquenessFilter::EdgeUniquenessFilterCursor::Shutdown() { input_cursor
 
 void EdgeUniquenessFilter::EdgeUniquenessFilterCursor::Reset() { input_cursor_->Reset(); }
 
+EmptyResult::EmptyResult(const std::shared_ptr<LogicalOperator> &input, const std::vector<Symbol> &symbols)
```

Comment:
```suggestion
EmptyResult::EmptyResult(std::shared_ptr<LogicalOperator> input, const std::vector<Symbol> &symbols)
```

Reply:
I followed what other logical operators' constructors look like, it is with const?